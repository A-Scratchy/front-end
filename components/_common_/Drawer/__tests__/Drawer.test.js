/* eslint-env jest */
import React from 'react';
import createSnapshotTest from 'test-utils/createSnapshotTest';

import Drawer from '../Drawer';

describe('Drawer', () => {
  it('should render with required props', () => {
    createSnapshotTest(<Drawer>Test</Drawer>);
  });

  it('should render with all props assigned', () => {
    createSnapshotTest(
      <Drawer className="test-class" isVisible>
        Test
      </Drawer>,
    );
  });
});
