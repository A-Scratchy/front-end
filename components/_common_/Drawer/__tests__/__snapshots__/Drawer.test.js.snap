// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drawer should render with all props assigned 1`] = `
<div
  className="test-class visible"
>
  <div
    className="content"
  >
    Test
  </div>
</div>
`;

exports[`Drawer should render with required props 1`] = `
<div
  className="hidden"
>
  <div
    className="content"
  >
    Test
  </div>
</div>
`;
